lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -61,25 +60,24 @@ def __del__(self):
     def __call__(self, notification):
         if isinstance(notification, ufe.AttributeValueChanged):
             if notification.name() == UsdGeom.Tokens.xformOpOrder:
-                mel.eval("evalDeferred(\"AEbuildControls\");")
+                mel.eval("evalDeferred -low \"refreshEditorTemplates\";")
 
     def onCreate(self, *args):
         ufe.Attributes.addObserver(self._item, self)
 
     def onReplace(self, *args):
         pass
 
-class MetaDataCustomControl(aecustom.CustomControl):
+class MetaDataCustomControl(object):
```

Comment:
I decided that we should not base our custom controls on the Maya one since it will try and get the plug from attribute (which is UFE here, so no plug). The base class was simple anyways - we don't need it.